lib/ansible/config/base.yml

```diff
@@ -1474,16 +1473,16 @@ INTERPRETER_PYTHON_DISTRO_MAP:
 INTERPRETER_PYTHON_FALLBACK:
   name: Ordered list of Python interpreters to check for in discovery
   default:
-  - /usr/bin/python
   - python3.9
   - python3.8
   - python3.7
   - python3.6
   - python3.5
+  - /usr/bin/python3
   - python2.7
   - python2.6
+  - /usr/bin/python
   - /usr/libexec/platform-python
-  - /usr/bin/python3
   - python
```

Comment:
Hmm, shouldn't plain `python3` be somewhere on this list?

Reply:
Potentially? I'm not sure why it was omitted previously.

Reply:
`python3` is kept out of this list to avoid picking up Python < 3.5.